packages/pn-personafisica-webapp/src/__test__/App.test.tsx

```diff
@@ -93,11 +90,10 @@ describe('App', () => {
     jest.restoreAllMocks();
   });
 
-
   /**
    * Tests che usano Component e inizializzazione "semplice" di i18n.
    */
-  describe("tests che non analizzano dettagli (test solo di renderizzazione)", () => {
+  describe('tests che non analizzano dettagli (test solo di renderizzazione)', () => {
```

Comment:
maybe we have to rework these tests. check the pa, paying attention to different behaviour between the two apps